### PR TITLE
Wick Bot dashboard

### PR DESCRIPTION
**Pull request by BeeXD**
Many of wick configuration are done in dashboard

### CODE REVIEW

File: websites/W/Wick Dashboard/presence.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+     else if (document.location.pathname.includes("/premium")) 
+      presenceData.details = "Viewing Wick Premium Plans";
+    
+  }
+
+  if (presenceData.details == null) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else 
+    presence.setActivity(presenceData);
```
Comment (EncryptedDev) [inline websites/W/Wick Dashboard/presence.ts:45]:
```suggestion
presence.setActivity(presenceData);
```

File: websites/W/Wick Dashboard/presence.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ browsingStamp = Math.floor(Date.now() / 1000),
+
+ title: any;
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "wick"
+  };
+
+  if (document.location.hostname == "wickbot.com") {
```
Comment (EncryptedDev) [inline websites/W/Wick Dashboard/presence.ts:14]: Not needed

File: websites/W/Wick Dashboard/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      "name": "BeeXD",
+      "id": "745199026021859338"
+    },
+    "service": "Wick Dashboard",
+    "description": {
+      "en": "Wick Bot Dashboard"
+    },
+    "url": "wickbot.com",
+    "version": "1.0.0",
+    "logo": "https://wickbot.com/wick.png",
```
Comment (EncryptedDev) [inline websites/W/Wick Dashboard/dist/metadata.json:13]: Must be on imgur

File: websites/W/Wick Dashboard/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+      "name": "BeeXD",
+      "id": "745199026021859338"
+    },
+    "service": "Wick Dashboard",
+    "description": {
+      "en": "Wick Bot Dashboard"
```
Comment (EncryptedDev) [inline websites/W/Wick Dashboard/dist/metadata.json:9]: Don't think this is the correct description 🤷‍♂️

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/W/Wick Dashboard/presence.ts (hunk 2 above)
Comment (BeeXD) [reply websites/W/Wick Dashboard/presence.ts:14]: what should i change it to?

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Fix CodeFactor issues

File: websites/W/Wick Dashboard/presence.ts (hunk 2 above)
Comment (Bas950) [reply websites/W/Wick Dashboard/presence.ts:14]: just remove

Comment (BeeXD) [reply websites/W/Wick Dashboard/presence.ts]: K

**Pull request closed without merging**